packages/ddex/README.md

```diff
@@ -1,7 +1,19 @@
 # Audius DDEX
 
-React app for managing Audius DDEX uploads.
+NodeJS server for managing Audius bulk ingestion (DDEX uploads). Serves a frontend for uploading DDEX files at the root path (source code in `../ddex-frontend`).
 
-* To test locally: make sure you are running `npm run sdk` at `~/audius-protocol` root. Then `npm run start:stage` or `npm run start:prod` in a separate process
-* To test in a more prod-like manner locally: `npm run stage:preview` or `npm run prod:preview`  
-* When ran on stage or prod, the app will query its local `/d_api/env` endpoint to get environment variables that are set on the node (because the React app can't access these directly for security reasons)
+### Local Dev
+`npm i && npm run dev:[stage|prod]`  
+
+If you want to run the frontend locally, you'll need to:
+1. Make sure you're running `npm run sdk` at `~/audius-protocol` root
+2. Start the frontend: `cd ../ddex-frontend && npm i && npm run start:[stage|prod]`
+3. If you want to test in a more prod-like manner, do `npm run [stage|prod]:preview` in the frontend directory.
+  * Then you can either visit http://localhost:4173/ddex
+  * OR visit the backend root path (i.e. `http://localhost:8926/ddex`) to see the frontend served as static assets from the backend. But to do this, you have to simulate the reverse proxy stripping the /ddex path prefix by commenting out `base` in the frontend `vite.config.js`. TODO: Growth area here to make it more seamless to have the backend serve the frontend locally
```

Comment:
this requires manually building the dist in /ddex-frontend for http://localhost:8926/ddex to display frontend assets when developing locally, right? should we add that as a step in these instructions?

Reply:
step 3 (`npm run [stage|prod]:preview`) builds the dist (maybe i should clarify in the instructions that that's what the step is doing). i imagine that flow won't actually happen anyway because because it's gonna be easier to just `npm run start` and build on that with hot reloading

Reply:
oops i totally missed the `npm run [stage|prod]:preview`. looks good as is